index.html

```diff
@@ -29,352 +32,374 @@
+                  <li><a href="" class="base-link-text2">css</a></li>
+                  <li><a href="" class="base-link-text3">javascript</a></li>
+                </ul>
+              </div>
+              <div class="base-button">
+                <a href="#">See Project</a>
+              </div>
+            </div>
           </div>
-          <div class="base-button flex-card">
-            <a href="">See Project</a>
+
+          <!----End Markup for Desktop styling 3-->
+          <div class="s3-container">
+            <div class="image-container3"></div>
+
+            <h2>Tonic</h2>
+            <div class="list-container">
+              <ul>
+                <li><a href="" class="text">CANOPY</a></li>
+                <li><img src="images/c1.png" alt="" /></li>
+                <li><a href="" class="text-2">Back End Dev</a></li>
+                <li><img src="images/c1.png" alt="" /></li>
+                <li><a href="" class="text-3">2015</a></li>
+              </ul>
+            </div>
+            <div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </div>
+            <div class="base-links">
+              <ul>
+                <li><a href="" class="base-link-text1">html</a></li>
+                <li><a href="" class="base-link-text2">css</a></li>
+                <li><a href="" class="base-link-text3">javascript</a></li>
+              </ul>
+            </div>
+            <div class="base-button flex-card">
+              <a href="">See Project</a>
+            </div>
           </div>
         </div>
-      </div>
-      <div id="section-4">
-        <!----------Begin mark for card-4---->
+        <div id="section-4">
+          <!----------Begin mark for card-4---->
 
-        <div class = "s4-container-dt">
-         
-          <div class = "card-text-4">
-      <h2>Uber Navigation</h2>
-      <div class = "list-container4-dt">
-        <ul>
-          <li><a href="" class="text">FACEBOOK</a></li>
-          <li><img src="images/c1.png" alt="image of a dot" /></li>
-          <li><a href="" class="text-2">Full Stack Dev</a></li>
-          <li><img src="images/c1.png" alt="image of a dot" /></li>
-          <li><a href="" class="text-3">2015</a></li>
-        </ul>
-      </div>
-      <div>
-      <p>
-        A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+          <div class="s4-container-dt">
+            <div class="card-text-4">
+              <h3>Uber Navigation</h3>
+              <div class="list-container4-dt">
+                <ul>
+                  <li><a href="" class="text">FACEBOOK</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-2">Full Stack Dev</a></li>
+                  <li><img src="images/c1.png" alt="" /></li>
```

Comment:
- [ ]  I noticed here the alt is empty,, kindly add a text describing the image for accessibility issues 🙏🏻

Reply:
Thank you, ma'am! I will attend to these issues at once.